README.md

```diff
@@ -23,15 +19,42 @@ PHP client library for [Coveralls](https://coveralls.io).
 
 # Installation
 
+tl;dr
+Try the phar file first, come back if it doesn't work for you.
+
+There are multiple ways to install php-coveralls. We recommend phar
+file installation.
+
+## Download phar file
+
+We started to create a phar file, starting from the version 0.7.0
+release. It is available at the URLs like:
+
+```
+https://github.com/satooshi/php-coveralls/releases/download/v0.7.0/coveralls.phar
+```
+
+Download the file and add exec permissions:
+
+```sh
+$ wget https://github.com/satooshi/php-coveralls/releases/download/v0.7.0/coveralls.phar
+$ chmod +x coveralls.phar
+```
+
+## Install by composer
+
 To install php-coveralls with Composer, run the following command:
 
 ```sh
 $ composer require satooshi/php-coveralls --dev
+# Call vendor/bin/coveralls
+$ vendor/bin/coveralls
```

Comment:
It is inconsistent, for phar you don't show how to run, for composer you show. No need to.